content/altinn-studio/v9/develop-a-service/process/pdf/_index.en.md

```
@@ -0,0 +1,300 @@
+    <bpmn:incoming>SequenceFlow_0c458hu</bpmn:incoming>
+    <bpmn:outgoing>SequenceFlow_5assd2s</bpmn:outgoing>
+</bpmn:serviceTask>
+```
+
+### Layout-set
+
+The PDF service task needs its own layout set to define the content. If you use the Arbeidsflyt-editor, Altinn Studio generates this automatically. You then only need to edit the content in `PdfLayout.json`.
+
+The files and folder structure should look approximately like this:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)